Sources/Purchasing/Package.swift

```diff
@@ -19,18 +19,31 @@ import Foundation
 ///
 @objc(RCPresentedOfferingContext) public final class PresentedOfferingContext: NSObject {
 
-    /// The identifier of the ``Offering`` containing this Package.
+    /// The identifier of the ``Offering`` containing this ``Package``.
```

Comment:
I'm late to the game here, but... what do I do with this class as a developer? Is it so I can use it for analytics? 
The docstring for the class doesn't really provide any context on it

Reply:
@aboedo Oops, missed this comment from yesterday! This is mainly for internal usage. Instead of passing `offeringIdentifier` from get offerings to post receipt, this objects gets passed around instead. It holds offering identifier, placement information, and targeting information.